Updated the arrow diagram

SamEdwardes opened the pull request:
Updated the arrow diagram:

- Updated to align with latest changes from #15
- Converted to drawio diagram to make it easier to maintain and change in the future

## New

![](https://github.com/sol-eng/bike_predict/blob/arrow-diagram-update/img/arrows.drawio.png?raw=true)

## Old

![](https://github.com/sol-eng/bike_predict/blob/arrow-diagram-update/img/arrows.png?raw=true)

edavidaja commented:
https://www.rstudio.com/images/RStudio-Brand-Guide-16June2021.pdf
https://www.rstudio.com/about/logos/

I don't think the "RStudio" in Connect is consistent with the brand guidelines.

SamEdwardes commented:
> https://www.rstudio.com/images/RStudio-Brand-Guide-16June2021.pdf https://www.rstudio.com/about/logos/
> 
> I don't think the "RStudio" in Connect is consistent with the brand guidelines.

Thanks David - I updated the diagram. I just removed the RStudio logo from connect and package manager since I could not find an official connect / package manager logo on the brand page.
